src/components/articles.css

```diff
@@ -0,0 +1,64 @@
+.articles__container {
+  margin-top: 1rem;
+}
+
+.articles-box {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 3rem;
+  background-color: #f6f6f6;
+}
+
+.articles-list {
+  margin: 0 1.5rem 0 1.5rem;
+  border-bottom: 3px solid #232a55;
+  border-radius: 0.2rem;
+  box-shadow: 0px 5px 10px #232a55;
+}
+
+.articles-list img {
```

Comment:
the articles go right up to the bottom of the page at the moment, it would look a bit nicer if there was a margin